apps/music/elements/music-search-results.js

```diff
@@ -59,105 +58,99 @@ var template =
 </style>
 <gaia-fast-list id="list">
   <template>
-    <li>
-      <a href="\${url}" data-file-path="\${name}" data-section="\${section}">
-        <img style="visibility: \${name ? 'visible' : 'hidden'}">
-        <h3>\${title}</h3>
-        <p>\${subtitle}</p>
-      </a>
-    </li>
+    <a href="\${url}" data-file-path="\${name}" data-section="\${section}">
+      <div class="image"><img></div>
+      <h3>\${title}</h3>
+      <p>\${subtitle}</p>
+    </a>
   </template>
 </gaia-fast-list>`;
 
 proto.createdCallback = function() {
-  var shadowRoot = this.createShadowRoot();
-  shadowRoot.innerHTML = template;
-
-  var $id = shadowRoot.getElementById.bind(shadowRoot);
+  this.innerHTML = template;
```

Comment:
I mentioned this to you prior on IRC. I found in some cases Gecko was dispatching really broken `scroll` events. I spent a very long time digging into this and eventually found pulling the list out of shadow-dom fixed this :/